spec/models/volunteer_spec.rb

```diff
@@ -24,4 +24,31 @@
     expect(volunteer.errors[:name][0]).to eq("can't be blank")
     expect(volunteer.errors[:about][0]).to eq("can't be blank")
   end
+
+  describe "Volunteer groups" do
```

Comment:
I'm not sure what this test is really testing. Since it's just a copy / paste of what's already in the Volunteers model, I'm not sure it's very valuable. I think I'd rather see a feature spec that has a user selecting a volunteer group from the dropdown, and verifying that it inserted the right group into the db.